src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
+    this.health = health;
+    this.name = 'Animal';
```

Comment:
[CODE QUALITY] The `name` property should be initialized with the `name` parameter passed to the constructor, not with the hardcoded string 'Animal'. Change this to `this.name = name;` to correctly assign the name of the animal.